test/testtokenize.cpp

```diff
@@ -7237,6 +7238,11 @@ class TestTokenizer : public TestFixture {
                                              "struct c<a<e, d...>, true> {};\n"));
     }
 
+    void noCrash7() {
+        ASSERT_NO_THROW(tokenizeAndStringify("enum : { };\n"
```

Comment:
It looks like garbage code. I suggest we move it to testgarbage. what do you think about throwing a syntax error for the code `enum : {`?